metadata/Cadastre/basic_parcels/utah_county_parcels_schema.md

```diff
@@ -0,0 +1,11 @@
+# Title
+
+Utah County Parcels Field Definitions
+
+## ID
+
+2aea10f4-92ba-4539-b473-456a0675db94
+
+### OWNERNAME
+
+<!--- No definition for this field --->
```

Comment:
Are we hoping that @rkelson provides this definition?

Reply:
my bad. I was looking for Notes at the end of the fields list. Due to it's position in the list it dates back at least decade when Debby Ames requested it be there. It is no longer maintained and could be removed but that is another can of worms so we'll just leave it.

Reply:
Rick sent me the field definitions for the unique fields, I'll be filling them in

> Are we hoping that @rkelson provides this definition?